### PR TITLE
Fix file path generation in configureTempto() method

### PR DESCRIPTION
**Pull request by kokosing**
Fix file path generation in configureTempto() method

Previously it was generating always constant name:
tempto-configuration-for-...yaml

Now it should generate something like:
tempto-configuration-for-kafka-abs34.yaml

### CODE REVIEW

**Comment by kokosing** [comment]
> 2021-09-13T10:14:29.1649330Z tests               | 2021-09-13 15:59:29 INFO: Reading configuration from tempto-configuration.yaml,/docker/presto-product-tests/conf/tempto/tempto-configuration-for-docker-default.yaml,/docker/presto-product-tests/conf/tempto/tempto-configuration-profile-config-file.yaml,/dev/null,/docker/presto-product-tests/conf/tempto/tempto-configuration-for-singlenode-ldap-and-file-1voll.yaml
> 2021-09-13T10:25:15.8590301Z tests               | 2021-09-13 16:10:15 INFO: Reading configuration from tempto-configuration.yaml,/docker/presto-product-tests/conf/tempto/tempto-configuration-for-docker-default.yaml,/docker/presto-product-tests/conf/tempto/tempto-configuration-profile-config-file.yaml,/dev/null,/docker/presto-product-tests/conf/tempto/tempto-configuration-for-hadoop-kerberos-cv7cu.yaml,/docker/presto-product-tests/conf/tempto/tempto-configuration-for-hadoop-kerberos-kms-uomvo.yaml

File: testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentContainers.java [hunk 1]
```diff
@@ -45,16 +55,27 @@ public static boolean isPrestoContainer(String name)
...
                     .withEnv("TEMPTO_CONFIG_FILES", temptoConfigFiles ->
                             temptoConfigFiles
                                     .map(files -> files + "," + temptoConfig)
                                     .orElse(temptoConfig));
         });
     }
+
+    private static String getParentDirectoryName(Path path)
+    {
+        checkArgument(path.getNameCount() > 2, "Parent directory name not found in: " + path);
```
Comment (losipiuk) [inline testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentContainers.java:72]: `>=`?

Comment (losipiuk) [reply testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentContainers.java:72]: `Cannot determine parent directory of ` + path ?

**Review by losipiuk — APPROVED** [review]

File: testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentContainers.java (hunk 1 above)
Comment (kokosing) [reply testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentContainers.java:72]: > >=?

Yes, it has to have a parent directory. So we expect two names here: `/parent.name/file.name`

Comment (losipiuk) [reply testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentContainers.java]: Yeah - two. Not necessarily more than two.

**Pull request merged**